lib/server.js

```diff
@@ -27,10 +27,7 @@ var Router = util.safeRequire('react-router');
 
 // a template of the `script` tag that gets
 // injected into the server rendered pages.
-var TEMPLATE = ['<script id="%s" type="application/javascript">var ',
-                Config.client.variableName,
-                ' = %s;</script>'
-              ].join('');
+var TEMPLATE = '<script id="%s" type="application/javascript">window._store = %s;</script>';
```

Comment:
please dont change the variable name to `_store`. client needs to read off of this variable. we are reusing the variable name https://github.com/paypal/react-engine/blob/master/lib/config.json#L5